File: Submissions/#0425_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    long n = sc.nextLong();
    long k = sc.nextLong();
    sc.close();

    long allTurn = n / k;

    if (allTurn % 2 == 1) {
      System.out.println("YES");
    } else {
      System.out.println("NO");
    }

  }

}

File: Submissions/#0425_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  long long n, k;
  cin >> n >> k;

  long long allTurn = n / k;

  if (allTurn % 2 == 1) {
    cout << "YES" << endl;
  } else {
    cout << "NO" << endl;
  }

}